babi/horizontal_scrolling.py

```diff
@@ -35,7 +35,7 @@ def scrolled_line(s: str, x: int, width: int) -> str:
 class _CalcWidth:
     @cached_property
     def _window(self) -> 'curses._CursesWindow':
-        return curses.newwin(1, 10)
+        return curses.newwin(10, 10)
```

Comment:
if this is overflowing the first line requiring more lines then there's a bug

Reply:
It was either this or actual tabs wouldn't re-render when changing the tab size.

Reply:
the height should not change, have you tried changing the width?

Reply:
That still fails.